src/store/reducers/appReducer.js

```diff
@@ -430,4 +434,6 @@ export const getActionPollingError = state =>
   getError(getAppReducer(state), epics.actionTypes.pollActionSettings);
 
 export const getActionPollingTimeout = state => getAppReducer(state).actionPollingTimeout;
+
+export const isDeployedAsOwner = state => getAppReducer(state).isDeployedAsOwner;
```

Comment:
Aren't all the other selectors named with "get" at the beginning?

Reply:
for boolean we have isDefaultLogo. I'm open to changing it, but since it return true/false isDeployedAsOwner makes more sense to me. Otherwise it would be getIsDeployedAsOwner

Reply:
I've renamed the variable so this is now a get